services/server/server.go

```diff
@@ -493,6 +496,70 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 	return res, nil
 }
 
+// SetSSHKey uploads public key.
+func (s *Server) SetSSHKey(ctx context.Context, in *serverpb.SetSSHKeyRequest) (*serverpb.SetSSHKeyResponse, error) {
+	//TODO: validate ssh key
+	var settings *models.Settings
+	err := s.db.InTransaction(func(tx *reform.TX) error {
+		var e error
+		if settings, e = models.GetSettings(tx); e != nil {
+			return e
+		}
+
+		username := "admin"
+		usr, err := user.Lookup(username)
```

Comment:
Note that there you start using `err` instead of `e`.
Most of our code looks like that:
```
	e := r.db.InTransaction(func(tx *reform.TX) error {
		// use err there
	}
```

Reply:
yeah, I'll rename.
In most cases we
```
e := r.db.InTransaction(func(tx *reform.TX) error {
		// use err there
	}
```
but in this file in other method

```
err := r.db.InTransaction(func(tx *reform.TX) error {
		// use e there
	}
```
Which one I should use or it doesn't matter?

Reply:
The most common one wins :)